mcs/class/corlib/System.Runtime.InteropServices/Marshal.cs

```diff
@@ -1897,5 +1897,84 @@ internal static void SetLastWin32Error (int error)
 			throw new NotSupportedException();
 		}
 #endif
+
+		internal struct MarshalerInstanceKey {
```

Comment:
Why not use ValueTuple?

Reply:
Didn't think of it. It's in corlib in all our profiles?

Reply:
It should be